src/inputs/forms/json-schema-form/helper.ts

```diff
@@ -32,6 +41,9 @@ export const initFormDataWithSchema = (schema?: JsonSchema, formData?: object):
     Object.keys(properties).forEach((key) => {
         const property = properties[key];
         result[key] = formData?.[key] ?? property.default ?? undefined;
+        if (property.type === 'array' && result[key]) { // array type needs conversion for component.
+            result[key] = cloneDeep(result[key]).map(d => ({ value: d }));
+        }
```

Comment:
```suggestion
        if (property.type === 'array' && result[key]) { // array type needs conversion for component.
           if (!Array.isArray(result[key]) result[key] = undefined;
           else {
             result[key] = result[key].map(d => ({ value: d }));
           }
        }
```
There is currently no requirement for an object to come in for each value in the array. Since each value of the array will have a primitive type, clone deep seems to cause unnecessary performance degradation..!

Also, how about guarding a case where an array is not entered in the default value of the schema?
For this purpose, I confirmed that a part that changes the selected to an array form when selected is not an array has been added to TextInput, but since the JsonSchemaForm component and the TextInput component must be viewed effectively separately, it would be good to have an appropriate guard here as well.

---

배열의 각 값에 객체가 들어오는 요구사항은 현재 없습니다. 배열의 각 값에는 모두 primitive 타입이 올 것이므로 clone deep은 불필요한 성능저하를 일으킬 것 같습니다..!

또한 스키마의 default 값에 array가 안 들어온 경우를 한 번 가드해주면 어떨까요?
TextInput에 이를 위해 selected가 array가 아닌 경우에 배열 형태로 바꿔주는 부분이 추가된 것을 확인했습니다만, JsonSchemaForm 컴포넌트와 TextInput 컴포넌트는 사실상 별개로 봐야하기 때문에, 여기에도 적절한 가드가 있으면 좋을 것 같습니다.